PWGCF/DataModel/FemtoDerived.h

```diff
@@ -245,14 +245,14 @@ DECLARE_SOA_COLUMN(Prong2Eta, prong2Eta, float);                    //! Track et
 DECLARE_SOA_COLUMN(Prong0Phi, prong0Phi, float);                    //! Track phi of charm hadron prong0
 DECLARE_SOA_COLUMN(Prong1Phi, prong1Phi, float);                    //! Track phi of charm hadron prong1
 DECLARE_SOA_COLUMN(Prong2Phi, prong2Phi, float);                    //! Track phi of charm hadron prong2
-DECLARE_SOA_COLUMN(CandidateSelFlag, candidateSelFlag, int8_t);     //! Selection of mass hypothesis for charm hadron (1 for Lc -> pkpi, 2 for Lc -> pikp, 4 for D+ -> pikpi)
+DECLARE_SOA_COLUMN(CandidateSelFlag, candidateSelFlag, int);        //! Selection of mass hypothesis for charm hadron (1 for Lc -> pkpi, 2 for Lc -> pikp, 4 for D+ -> pikpi)
```

Comment:
I believe that `int8_t` is the only integral type affected by this issue. Why not use `int16_t`?

Reply:
Sorry, but I couldn't find a line where additional casting would be needed. Can you refer them?

Reply:
I don't see why `charmHadMcSel` has to be `int`.

Reply:
flagMc was `int_8`, so Configurable in charm femto task is:
`Configurable<int> charmHadMcSel{"charmHadMcSel", DecayChannelMain::LcToPKPi, "charm hadron selection for mc, DplusToPiKPi = 1, LcToPKPi = 17"};`
here caste Configurable `int` to `int8_t`
Filter hfMcSelFilter = nabs(aod::fdhf::flagMc) == static_cast<int8_t>(charmHadMcSel.value);
It looks good, but it always takes the default value casted to int8_t, So that Configurable is useless. 
Then the directly way is: align `flagMc` also to `int`. I think this issue also reported from someone in the O2 Analysis channel before

Reply:
Sorry but that does not answer my question.

Reply:
sorry, I don't know why it should be `int16_t`? can you explain more?

Reply:
> Your solution is to promote all concerned variables to `int`. I am proposing to make them the smallest working integral type, which AFAIK is `int16_t` .

 I agree with your point and I also want to improve it with this point. However as I mentioned above, I already tried `int16_t`. It doesn't work (seems not support). So could you please approve it with current version?

Reply:
I already tried both case. it is not the case.